src/main/java/world/jnc/invsync/permission/PermissionRegistry.java

```diff
@@ -66,6 +66,11 @@ public void registerPermissions() {
           PlayerSerializer.spiceOfLifeSyncModule.getPermission(),
           "Allow this user's Spice of Life food list to be synchronized",
           PermissionDescription.ROLE_USER);
+    if (PlayerSerializer.cyclicSyncModule.canBeEnabled())
```

Comment:
Can you please add it in alphabetical order?

Reply:
The other mods aren't in alphabetical order. So I took a guess at what you meant.